webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -181,10 +188,16 @@ export class MembraneWebRTC {
     });
 
     await phoenix_channel_push_result(this.channel.join());
-    const { maxDisplayNum } = await phoenix_channel_push_result(
-      this.channel.push("start", {})
-    );
-    this.maxDisplayNum = maxDisplayNum;
+
+    try {
+      const { maxDisplayNum } = await phoenix_channel_push_result(
+        this.channel.push("start", {})
+      );
+      this.maxDisplayNum = maxDisplayNum;
+    } catch (e) {
+      this.callbacks.onServerError?.(e);
+      this.stop();
+    }
```

Comment:
Do we need this `try` and `onServerError`? Shouldn't we use `onConnectionError` somehow? `onServerError` looks very similar to `onConnectionError` and I think that `onConnectionError` is also invoked when there is any problem on the server side. @Qizot?